book/src/pong-tutorial/contribution.md

```diff
@@ -0,0 +1,13 @@
+# Call for contribution
+
+And... that's where the pong chapter ends.
+Amethyst is still in development and writing good tutorials is hard, so the guide is not complete yet.
+We are sorry this might feel a bit disappointing.
+
+Fortunately, the pong example is complete, and you can find the entire code with balls, score and music on the example pages available [here](https://www.amethyst.rs/doc). You can get additional help by leaving a post on [our subreddit](https://www.reddit.com/r/Amethyst) or on our [Discord server](https://discord.gg/GnP5Whs).
+
+We hate to have an incomplete tutorial as much as you do, and you can help us complete it!
+As stated above, the code is ready, tested and working. The only thing we need now is good enough English.
```

Comment:
The only thing we need now is good enough English

Sounds a bit weird imo

Reply:
Does it?

Reply:
"The only thing we are in need" sounds weirder to me

Reply:
Honestly, @Moxinilian it sounded a bit strange to me, too. I  think you can leave that sentence out. We have enough people with that skill, the missing resources are either time or engine knowledge.